lib/entities/app-action.ts

```diff
@@ -70,6 +70,7 @@ export type AppActionProps = AppActionCategory & {
    * Human readable description of the action
    */
   description?: string
+  type?: 'endpoint' | 'function'
```

Comment:
So endpoint is basically when it’s a URL it’s sending a request to and function when it’s a CF hosted function?